diagnostic_aggregator/src/aggregator_node.cpp

```diff
@@ -34,34 +34,20 @@
 
 /**< \author Kevin Watts */
 
-#include <exception>
+#include <rclcpp/rclcpp.hpp>
```

Comment:
Please include <memory> for using `make_shared`

Reply:
Are you referring to the original try-catch?

Reply:
Sorry, the github syntax swallowed parts of the message.
I was asking to `#include <memory>` because you introduce `make_shared`.